packages/frontend/src/pages/scaling-projects/props/getProjectHeader.ts

```diff
@@ -37,6 +39,8 @@ export function getProjectHeader(
         : undefined,
     purpose: project.display.purpose,
     technology: project.technology.category,
+    ratingEnabled: config.features.rating,
```

Comment:
can we do it like in config? something like this will do this:

```
diff --git a/packages/frontend/src/pages/scaling-projects/props/getProjectHeader.ts b/packages/frontend/src/pages/scaling-projects/props/getProjectHeader.ts
index 61ffce82..a3149196 100644
--- a/packages/frontend/src/pages/scaling-projects/props/getProjectHeader.ts
+++ b/packages/frontend/src/pages/scaling-projects/props/getProjectHeader.ts
@@ -39,8 +39,7 @@ export function getProjectHeader(
         : undefined,
     purpose: project.display.purpose,
     technology: project.technology.category,
-    ratingEnabled: config.features.rating,
-    ratingEntry: project.rating,
+    ratingEntry: config.features.rating && project.rating,
   }
 }
 
diff --git a/packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx b/packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx
index 15957f84..a3a469d8 100644
--- a/packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx
+++ b/packages/frontend/src/pages/scaling-projects/view/ProjectHeader.tsx
@@ -22,8 +22,7 @@ export interface ProjectHeaderProps {
   transactionMonthlyCount?: string
   purpose: string
   technology: string
-  ratingEnabled?: boolean
-  ratingEntry?: Layer2Rating
+  ratingEntry?: false | Layer2Rating
 }
 
 export function ProjectHeader(props: ProjectHeaderProps) {
@@ -67,7 +66,7 @@ export function ProjectHeader(props: ProjectHeaderProps) {
     },
   ]
 
-  if (props.ratingEnabled) {
+  if (props.ratingEntry) {
     const ratingStat = {
       title: 'Stage',
       value: props.ratingEntry ? (

```